### PR TITLE
docs: requested improvements regarding icmBaseURL + formatting improvements

### PR DESCRIPTION
**Pull request by shauke**
## Other Information

[AB#91858](https://dev.azure.com/intershop-com/cefd1005-00a7-4c79-927f-a16947d1b2e6/_workitems/edit/91858)

### CODE REVIEW

**Review by SGrueber — DISMISSED** [review]

File: docs/concepts/hybrid-approach.md [hunk 1]
```diff
@@ -81,7 +82,7 @@ The server-side rendering process must be started with `SSR_HYBRID=1`.
 
 > :warning: **Only for development environments**: The PWA must also be run with secure URLs if NGINX to handle `https` access to the PWA is not deployed.
```
Comment (mglatter) [inline docs/concepts/hybrid-approach.md:83]: Sentence may need rephrasing. 
Check formatting of :warning: (also in line 87)

**Comment by mglatter** [comment]
Review to be continued tomorrow for the remaining .md files. I'll inform you when done.

**Comment by mglatter** [comment]
Review to be continued...

File: docs/guides/getting-started.md [hunk 2]
```diff
@@ -73,9 +123,9 @@ Almost everything can be typed correctly without using `any`.
 If generic typing is not possible, you can use `unknown` or `object` for utility functions.
 In most other cases, the typings can be correctly applied.
 
-We encourage everybody to up their game about strict typing with TypeScript, but as always, there are ways around it.
-We supply a [`tsconfig.app-no-checks.json`](../../tsconfig.app-no-checks.json) in the source root, that can be used for Angular builders in the `angular.json` to disable all safety features.
-However, we don't support doing this for production builds.
+We encourage everybody to improve their skills in strict typing with TypeScript, but as always, there are ways around it.
```
Comment (mglatter) [inline docs/guides/getting-started.md:126]: I would actually delete the second part of this sentence ", but as always, there are ways around it."

I also suggest deleting ll. 118 - 120 beginning with "Learning  the TypeScript ..." and ending with "... nobody will be the wiser." Doesn't match our tone of voice and is giving an opinion which is to be avoided. I also don't see any point in providing this kind of information. If I'm wrong, please rephrase this section.

Comment (shauke) [reply docs/guides/getting-started.md:126]: Changed as suggested.

File: docs/guides/upgrading-pwa.md [hunk 3]
```diff
@@ -7,40 +7,40 @@ kb_sync_latest_only
 
 # Upgrading Dependencies
```
[the remark this answers is not in the document]
Comment (shauke) [reply docs/guides/upgrading-pwa.md:8]: renamed all relevant places from `upgrade` to `update`

File: docs/guides/getting-started.md (hunk 2 above)
Comment (mglatter) [reply docs/guides/getting-started.md:126]: What about deleting the second part of this sentence ", but as always, there are ways around it."?

**Pull request merged**